libs/utils/utils.js

```diff
@@ -236,7 +272,13 @@ export const [setConfig, updateConfig, getConfig] = (() => {
       config = { env: getEnv(conf), ...conf };
       config.codeRoot = conf.codeRoot ? `${origin}${conf.codeRoot}` : origin;
       config.base = config.miloLibs || config.codeRoot;
-      config.locale = pathname ? getLocale(conf.locales, pathname) : getLocale(conf.locales);
+      if (conf.languages) {
+        config.locale = pathname
+          ? getLanguage(conf.languages, conf.locales, pathname)
```

Comment:
is `pathname` check needed? I see your `getLanguage` method has `pathname = window.location.pathname` by default.

Reply:
yes, the pathname check is needed for use cases like in the region-nav .
@Ruchika4 will be using it so she uses the same logic as utils for getting the lang & region from a url that she passes as a parameter.